quickstart/resources/views/tasks/index.blade.php

```diff
@@ -0,0 +1,26 @@
+@extends('layouts.app')
+
+@section('title')
+    <title>{{ __('task.task') }}</title>
+@endsection
+
+@section('content')
+<div class="card mt-5">
+    <div class="card-header">
+        {{ __('task.new_task') }}
```

Comment:
```suggestion
        {{ trans('task.new_task') }}
```
sửa các chỗ tương tự